admin_app/src/components/ContentCard.tsx

```diff
@@ -72,7 +74,7 @@ const ContentCard = ({
             year: "numeric",
             hour: "numeric",
             minute: "numeric",
-            hour12: true,
+            hour12: false,
```

Comment:
Change this to false because I was finding weird hours such as 0:15pm. Not sure if it exists. We should look for a fix!

Reply:
haha fair